client/js/verification-portal/community-health-center/form.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react'
+import { reduxForm } from 'redux-form'
+import injectSheet from 'react-jss'
+
+import RegisterField from '../components/register-field'
+import Button from '../../components/button'
+import Spacer from '../../components/spacer'
+
+const registerFields = inputField => {
+	if (!inputField.name) return inputField.map(field => registerFields(field))
+	return <RegisterField inputField={inputField} key={inputField.name} />
+}
+class chcForm extends Component {
```

Comment:
I'm pretty sure this doesn't need to be a class, you can just return the form and it will render.